src/logic/PlayerController.cpp

```diff
@@ -103,11 +103,12 @@ PlayerController::PlayerController(World::WorldInstance& world,
 void PlayerController::onUpdate(float deltaTime)
 {
     m_RefuseTalkTime -= deltaTime;
-    // This vob should react to messages
-    getEM().processMessageQueue();
 
     m_AIStateMachine.doAIState(deltaTime);
 
+    // This vob should react to messages
+    getEM().processMessageQueue();
```

Comment:
What impact does switching the order of `processMessageQueue` and `doAIState` have? Does it fix a bug?